package.json

```diff
@@ -28,7 +28,12 @@
     "mac": {
       "artifactName": "jitsi-meet.${ext}",
       "category": "public.app-category.video",
-      "darkModeSupport": true
+      "darkModeSupport": true,
+      "hardenedRuntime": false,
+      "extendInfo": {
+        "NSCameUsageDescription": "jitsi meet requires access to your camera in order to make video-calls.",
+        "NSMicrophoneUsageDescription": "jitsi meet requires access to your microphone in order to make calls (audio/video)."
```

Comment:
Camera in the key is misspelled, it should be `NSCameraUsageDescription`. Also, `jitsi meet` should be capitalized.

```
"NSCameraUsageDescription": "Jitsi Meet requires access to your camera in order to make video-calls.",
"NSMicrophoneUsageDescription": "Jitsi Meet requires access to your microphone in order to make calls (audio/video)."
```

Reply:
Thanks, has been fixed in the original, will close and send a new request